Jenkinsfile

```diff
@@ -526,6 +526,10 @@ pipeline {
         Smoke_targets = "check MIOpenDriver"
         NOCOMGR_flags   = " -DMIOPEN_USE_COMGR=Off"
     }
+    triggers{
+        //triggers periodic runs midnight everyday
+        cron(env.BRANCH_NAME == "miopen_nightly" ? '0 0 * * *' : '')
```

Comment:
Can we take the branch name from the environment? Also, I believe the schedule can be part of the Jenkins config as well